files/nrpe/check_openstack.py

```diff
@@ -308,15 +315,17 @@ def delete_orphaned_floating_ips(self):
     for tenant_ip in self.nova.floating_ips.list():
       self.nova.floating_ips.delete(tenant_ip)
     if len(self.nova.floating_ips.list()) != 0:
-      logging.warn('All floating IPs of instance creation test tenant were not deleted.') 
+      logging.warn('All floating IPs of instance creation test tenant were not deleted.')
 
   def execute(self):
     results = dict()
     try:
       self.delete_orphaned_instances()
       results['10_delete_instance_ms'] = self.time_diff()
-      self.delete_orphaned_floating_ips()
-      results['20_delete_floatingip_ms'] = self.time_diff()
+      if self.options.no_ping == False:
+        # TODO THIS DOES PROBABLY NOT WORK IN cPOUTA
```

Comment:
Is the general idea that this TODO will be checked "later"?

I'm not sure why it wouldn't work in cPouta, but if there are concerns IMO they should be addressed before we can merge

Reply:
Removed the TODO.